lottie/src/main/java/com/airbnb/lottie/LottieDrawable.java

```diff
@@ -866,7 +866,7 @@ public boolean isAnimating() {
     return animator.isRunning();
   }
 
-  void setSystemAnimationsAreEnabled(Boolean areEnabled) {
+  public void setSystemAnimationsAreEnabled(Boolean areEnabled) {
```

Comment:
Can you leave this visibility the same as before?

Reply:
I guess it is possible, but then this functionality will be accessible only via XML attribute. I think it's a good idea to give ability to override this behaviour from the code (conditionally, for instance) as well.
Alternatively, I can add `setSystemAnimationsAreEnabled` to LottieAnimationView and leave `setSystemAnimationsAreEnabled` of LottieDrawable as it was. 
@gpeal thoughts?

Reply:
@IljaKosynkin If you want a programmatic API (not a bad idea), then there should be one on LottieAnimationView as well (in addition to the attr).

And in that case, it's important to make all 3 have similar names. Let's go with `lottie_ignoreDisabledSystemAnimations` and `setIgnoreDisabledSystemAnimations(boolean)`

It's a bit verbose but it's fairly clear I think.

Reply:
Ok, just to make sure @gpeal, you suggest to rename attribute to `lottie_ignoreDisabledSystemAnimations` and adding two public functions to both LottieAnimationView and LottieDrawable with name `setIgnoreDisabledSystemAnimations`, which will set a flag to ignore system settings in LottieDrawable. 
I also to leave visibility of `setSystemAnimationsAreEnabled` in LottieDrawable as package-private.
Everything is correct?

Reply:
Yes, and add a javadoc to both new APIs.

Reply:
@gpeal updated the PR, sorry for the delay!
I'm not sure I got right splitting of flags in LottieDrawable, it spanned a little bit further than I originally wanted it to, so let me know if everything is alright.